llumnix/llm_engine_manager.py

```diff
@@ -227,10 +231,14 @@ async def _push_migrations(self) -> None:
 
     async def _migrate(self, pair_migration_type: PairMigrationConstraints, migrate_in_num_requests: int) -> None:
         async def migrate_done_callback(ret, migrate_instance_pair: Tuple[str, str]) -> None:
-            if migrate_instance_pair[0] in self.instance_migrating:
-                self.instance_migrating[migrate_instance_pair[0]] = False
-            if migrate_instance_pair[1] in self.instance_migrating:
-                self.instance_migrating[migrate_instance_pair[1]] = False
+            if pair_migration_type != PairMigrationConstraints.PREFILL_2_DECODING:
+                migrate_out_instance_id, migrate_in_instance_id = migrate_instance_pair
+                self.num_migrating_pair -= 1
+                if migrate_out_instance_id in self.inflight_migrating:
+                    self.inflight_migrating[migrate_out_instance_id][0] -= 1
```

Comment:
Does 0 indicate migrate_out and 1 indicate migrate_in? Is this list only of length 1? Can we use other structure or add an annotation?